examples/stimuli/tracker_dealer.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+======================
+Tracker Dealer Example
+======================
+
+This file shows how to interleave multiple Tracker objects using
+:class:`expyfun.stimuli.TrackerDealer`.
+
+In this case, a modeled human subject generates two curves (one for each trial
+type: 1 & 2).
+
+@author: maddycapp27
+"""
+
+import numpy as np
+from expyfun.stimuli import TrackerUD, TrackerDealer
+from expyfun.analyze import sigmoid
+import matplotlib.pyplot as plt
+
+# define parameters of modeled subject (using sigmoid probability)
+true_thresh = [30, 40]
+slope = 0.1
+chance = 0.5
+
+# define parameters for each tracker (assuming each tracker uses same rules)
+up = 1
+down = 1
+step_size_up = [9, 3]
+step_size_down = [3, 1]

```

Comment:
yeah, a comment would be good (since initially I thought they were different values that were going to be fed into the two different `TrackerUD`s that were getting created).